router/builders.go

```diff
@@ -131,6 +132,14 @@ func buildHTTPOutput(sourceSettings *OutputSettings) (*outputs.HTTPClient, error
 		return nil, fmt.Errorf("http action requires a method to be specified")
 	}
 
+	if len(sourceSettings.BodyFile) > 0 && len(sourceSettings.BodyContent) > 0 {
+		return nil, fmt.Errorf("http action requires only supports body-file or body-content, not both")
+	}
+
+	if len(sourceSettings.BodyFile) <= 0 && len(sourceSettings.BodyContent) <= 0 {
+		log.Println("no body specified with http action, will send empty body")
```

Comment:
request without body is pretty legit. May lead to log pollution

Reply:
right - I had originally put it in to be part of the debug log trace but until we have real levelled logging in place, this can wait. ill take it out.